src/utils.js

```diff
@@ -23,10 +23,10 @@ export const appendQueryParams = (queryParams, url) => {
   return url + separator + urlEncode(queryParams)
 }
 
-export const appendQueryStringArrayParams = (key, values, url) => {
-  const separator = contains('?', url) ? '&' : '?'
+export const appendQueryParamsAsArray  = (key, values, url) => {
+  const separator = contains('?', url) ? '' : '?'
```

Comment:
Why did this line change? Don't we still want to add a `&` if there is already a `?` present in the URL?